pkg/controller/nodekeeper/statuschanged_predicate.go

```diff
@@ -0,0 +1,37 @@
+package nodekeeper
+
+import (
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+)
+
+var StatusChangedPredicate = predicate.Funcs{
```

Comment:
This isn't a `StatusChanged` predicate. More a 'ignore master' predicate? :joy: 

Does this get called for initial reconcile?

Reply:
Initial reconcile calls Create. Have added back into predicates.